### ISSUE TITLE
New language has no flag displayed

### ISSUE THREAD

**Issue by pongraczi** [laravel-enso/Enso#51]
When I add a new language, I got no flag displayed for it, even the flag files are exist and the database contains similar entries for old and new languages.

![kijeloles_658](https://user-images.githubusercontent.com/5891983/36567505-f0119c52-1826-11e8-9cb7-68845e7e2cdb.png)

I found that, two files contains path to the flags, but only for the default languages, seems never updated:
* public/js/enso.js
* public/js/enso.js.map

Did I miss something?

**Comment by aocneanu** [comment]
This issue belongs to `laravel-enso/localisation` repo.

- Did you checked the [readme](https://github.com/laravel-enso/Localisation/blob/master/README.md)?

- Did you checked the previous issues on this subject?

>even the flag files are exist
- Where did you placed the new flag?

You can also contribute with Magyar localisation for the global project.

https://github.com/laravel-enso/Localisation/pulls?q=is%3Apr+is%3Aclosed

**Comment by pongraczi** [comment]
Thank for pointing to the readme of localisation, I missed it.  
Otherwise, after I added a new language, I was able to translate using the localization manager and I got several messages displayed in this new language.  
Regarding flags, I did nothing, there are flags under public/images/vendor/flag-icon-css/flags/1x1 and 4x3.
Using Enso 2.0.35 it just works, I got the flag immediately, without any installation.
